packages/manager/modules/anthos/src/anthos.html

```diff
@@ -0,0 +1,76 @@
+<oui-header data-heading="{{ ::'anthos_tenants_title' | translate }}">
+    <oui-guide-menu data-text="{{ ::$ctrl.guides.title }}">
+        <oui-guide-menu-item
+            data-ng-repeat="guideItem in $ctrl.guides.list track by guideItem.url"
+            data-href="{{ ::guideItem.url }}"
+            data-external="guideItem.external"
+            ><span data-ng-bind="::guideItem.name"></span>
+        </oui-guide-menu-item>
+        <oui-guide-menu-divider></oui-guide-menu-divider>
+        <oui-guide-menu-item
+            data-href="{{ ::$ctrl.guides.footer.url }}"
+            data-external="$ctrl.guides.footer.external"
+            ><span data-ng-bind="::$ctrl.guides.footer.name"></span>
+        </oui-guide-menu-item>
+    </oui-guide-menu>
+</oui-header>
+
+<oui-datagrid id="hpcAnthosTenantsDatagrid" data-rows="$ctrl.tenants">
+    <oui-datagrid-column
+        title=":: 'anthos_tenants_grid_headers_serviceName' | translate"
+        data-property="serviceName"
+        data-type="string"
+        data-searchable
+        data-filterable
+        data-sortable="asc"
+    >
+        <a
+            data-ng-href="{{:: $ctrl.getServiceNameLink($row) }}"
+            data-ng-bind=":: $value"
+        ></a>
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'anthos_tenants_grid_headers_displayName' | translate"
+        data-property="displayName"
+        data-type="string"
+        data-searchable
+        data-filterable
+        data-sortable
+    >
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'anthos_tenants_grid_headers_location' | translate"
+        data-property="location"
+        data-type="string"
+        data-searchable
+        data-sortable
+        data-filterable
+    >
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'anthos_tenants_grid_headers_version' | translate"
+        data-property="version"
+        data-type="string"
+        data-sortable
+        data-filterable
+    >
+    </oui-datagrid-column>
+    <oui-datagrid-column
+        data-title=":: 'anthos_tenants_grid_headers_vrackId' | translate"
+        data-property="vrackId"
+        data-type="number"
+        data-sortable
+        data-filterable
+    >
+    </oui-datagrid-column>
+    <oui-datagrid-topbar>
+        <button
+            type="button"
+            class="oui-button oui-button_secondary oui-button_icon-left"
+            data-ng-click="$ctrl.createTenant()"
```

Comment:
We do not have this button in the mocks

Reply:
I anticipate it because we will add it into next version.
But true it's not in mock, i have to no add it.

Removed.